### DISCUSSION TITLE
Plan for Final Submission

### DISCUSSION THREAD

**Discussion by ryanpseely** [ecn310/course-project-nepobabies#16] [General]
## NepoBabies Report Finalizing Assignments

| Task | Target Date | Completed Date | Assignee | Notes |
|---|---|---|---|---|
| Submit whole package | Dec. 18 |   | Ryan | Still need to decide where to host documentation |
| Final proofread of report | Dec. 17 | | R&R | Will use grammarly (or the writing center) |
| Final proofread of documentation | Dec. 17 | | R&R | Will use grammarly (or the writing center) |
| Final documentation testing by outsider | Dec. 16 | | Ryan  | |
| Edit report | Dec. 16 | | Rachel | |
| Deliver report to outside reader | Dec. 15 | | Ryan | Ryan's Mom |
| Preliminary documentation testing by outsider | Dec. 14 | | RR asks Allie(?) | |
| Documentation testing by insider | Dec. 14 | | Ryan | |
| Revise report | Dec. 14 | | R&R | |
| Finalize documentation for testing | Dec. 13 | | R&R | |
| Revise documentation | Dec. 12 | | R&R | |
| Submit final draft of analysis | Dec. 12 | | Ryan | Due by 8 a.m. to Prof. Buzard |
| First draft of report | Dec. 11 | | R&R | |
| Execute documentation plan  | Dec. 11 | | R&R | |
| Finalize abstract / intro | Dec. 10  | | Rachel | |
| Finalize discussion / conclusion | Dec. 10  | | Ryan | |
| Finalize data section | Dec. 9  | | Rachel| |
| Present in class for feedback | Dec. 8  | | R&R | |
| Finalize analysis section | Dec. 8  | | R&R | |
| Finalize hypothesis/theory section | Dec. 7  | | Ryan | |
| Finalize lit review and references | Dec. 6  | | Rachel | |
| Complete analysis | Dec. 6  | | R&R | all figures/tables need to be done |

**Comment by ryanpseely** [comment]
### Documentation Plan
#### December 6, 2023
Rachel and I are creating a plan for organizing and submitting our documentation!
- We plan to create the high level README on GitHub. This should be completed by December 16. 
- Analysis outputs will be attached to GitHub by December 11.
- All graphs in .png format on GitHub. All graphs should be uploaded to GitHub by December 10. 
- Do-files on GitHub (separate them into data exploration, data analysis, and unemployment rate data input). These files should be split, organized and uploaded by December 9.
- the original dataset contains too many variables to go on GitHub. It has been stored on OneDrive. The cleaned dataset (only variables we need) is stored on GitHub.
- Place figures/tables in data appendix by December 13.

**Reply by kbuzard** [reply]
> the original dataset contains too many variables to go on GitHub. It has been stored on OneDrive. The cleaned dataset (only variables we need) is stored on GitHub.

Make sure you have the code that shows how you cut down and created this file.

> We plan to create the high level README on GitHub.

Don't forget putting something in the Data Appendix, or it might even just be a footnote in the data section if you just need to give a link to where everything is. Don't forget to make your repo public.

Where are you going to describe how someone should go through the analysis step by step, including downloading the data and codebook(s)?
